pkilint/rest/__init__.py

```diff
@@ -94,3 +94,21 @@ def certificate_lint(linter_group_name: str, linter_name: str, doc: model.Certif
     parsed_doc = doc.parsed_document
 
     return linter_instance.lint(parsed_doc)
+
+@app.get('/ocsp/linter')
+def ocsp_linter_validations() -> List[model.Validation]:
+    """Returns the set of validations performed by the OCSP response linter"""
+
+    linter = ocsp.create_ocsp_response_linter()
+
+    return linter.validations
+
+@app.post('/ocsp')
+def ocsp_response_lint(doc: model.OcspResponseModel) -> model.LintResultList:
+    """Lints the specified OCSP response"""
+
+    linter = ocsp.create_ocsp_response_linter()
```

Comment:
Ideally, the linter is initialized once on server startup in the same manner that the TLS BR and SMIME BR validators are instantiated once and reused. This provides a performance benefit, as the individual validators need not be recreated for each request. Some of the validators require file I/O, so it's nice to avoid that. Additionally, reusing validators is safe to do, as they are stateless.

Let me know if you prefer that I tweak this logic to mirror how the TLS and SMIME BR validators are initialized and invoked per request, or if you prefer to make the change.

Reply:
Thank you for the feedback. I have added a commit now that makes sure the ocsp linter is only initialized one. I agree it makes more sense. I did not use linter groups since there is currently only one linter for OCSP. Let me know if you want to change this.

Reply:
Thanks, @mans-andersson. I think for now we're good with not using linter groups for OCSP responses. We can revisit that decision if/when support for the CA/B Forum (or other standards group) OCSP response profile is added.